### PR TITLE
Please read this

### PR DESCRIPTION
**Pull request by dbpokorny**

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing :+1:`:+1:`, or :-1:`:-1:`.
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both :+1:up- and :-1:down-votes are disregarded.
Comments containing monkeys are disregarded. :monkey: :monkey_face: :see_no_evil: :hear_no_evil: :speak_no_evil:
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **7** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:*
 bce285d84b210a841759779bd968791b2a03e74a

**Comment by dos1** [comment]
tl;dr :+1:

**Comment by dbpokorny** [comment]
no no no, downvote this one, I want to make an ascii art tombstone

**Comment by samis** [comment]
:+1:

**Comment by plor** [comment]
:+1:

**Comment by drocta** [comment]
50 years?

If code to do this gets added, and then later gets removed, what happens?
Is this meant to be a legal document, or just a statement of intent?

**Comment by dbpokorny** [comment]
@drocta these are great questions

**Comment by drocta** [comment]
BTW, can't the PR author remove a pull request themselves?

**Comment by dbpokorny** [comment]
people can choose if they want to think about the anythingbot's future or bury it, and commit to a course of action one way or the other. I'm suggesting burial while acknowledging the potential for a profitable future. The main question on my mind right now is: what does the anythingbot want?

Hello pullrequestlive.com!

**Comment by SpenserJ** [comment]
Why would burying it be a logical course of action? Why not continue to iterate on what we have, or split into multiple projects? I see no reason to slap a "discontinued" badge on the bot.

**Comment by dbpokorny** [comment]
I feel like the video feed is a problem that needs to be solved. I'm happy to solve it; I don't know what to do about it. Can we just let anyone queue up a video at botwillacceptanything.com:3000/queue? I would much rather have the bot play what web visitors want it to play than what it is doing now.

**Comment by SpenserJ** [comment]
I think that allow users to queue tracks up would be neat. You might even be able to repurpose some of the voting code so that users could vote out a video, if it is considered inappropriate. I don't think it should require a vote to add a video though.

**Comment by drocta** [comment]
I agree that the ability to add tracks to the queue would be cool.

**Comment by clay-to-n** [comment]
:-1:

**Comment by drocta** [comment]
Well, the next person to vote will push it above 7 votes, and either put it above or below 65%.

It's been significantly over 15 minutes.

So, whoever votes next may very well decide whether this gets added.

Personally I am against it.
It seems odd to me that the PR submitter cannot change their vote to neutral, but oh well.

**Comment by PurHur** [comment]
Because its no great change and not well-thought-out :-1:

**Comment by Prior99** [comment]
Thank you but I did more trolling then actually helping for this project. I believe all funds added to this projects should be given to the electronic frontiert foundation or any other charity organization or we should use it to do useless funny things with it in the name of the bot, so :-1:
